src/html/htmprint.cpp

```diff
@@ -709,8 +709,11 @@ bool wxHtmlEasyPrinting::DoPreview(wxHtmlPrintout *printout1, wxHtmlPrintout *pr
     }
 
     wxPreviewFrame *frame = new wxPreviewFrame(preview, m_ParentWindow,
-                                               m_Name + _(" Preview"),
-                                               wxPoint(100, 100), wxSize(650, 500));
+                                               wxString::Format(_("%s Preview"), m_Name),
+                                               wxPoint(100, 100),
```

Comment:
Probably should use `wxDefaultPosition` here.

Reply:
I thought that too when I saw it centering the window in the next line, but left it alone for fear of appearing too nit-picky :)